src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -159,44 +160,34 @@ const JobHistory = _.flow(
           },
           columns: [
             {
-              size: { basis: 600, grow: 0 },
+              size: { basis: 500, grow: 0 },
               headerRenderer: () => h(HeaderCell, ['Job']),
               cellRenderer: ({ rowIndex }) => {
                 const {
-                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId, workflowStatuses,
-                  status, submissionEntity
+                  methodConfigurationNamespace, submitter, submissionId, workflowStatuses
                 } = submissions[rowIndex]
                 return h(Fragment, [
                   div([
                     div([
                       methodConfigurationNamespace !== namespace && span({ style: styles.deemphasized }, [
                         `${methodConfigurationNamespace}/`
                       ]),
-                      methodConfigurationName
+                      link({
+                        target: '_blank',
```

Comment:
I think we should make it clearer that this link is to view job details. You could add a tooltip to each method config link, or add a note to the column header e.g. "Job (click for details)".